recipes/interface_meta/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "interface_meta" %}
+{% set version = "1.1.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 68bcafb9dd659f0043ef4d61f2bf86f84114a49423777bfec98fdf46174e2c00
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - decorator
```

Comment:
Are you sure you need all of these dependencies here? Usually, only `pip` and `python` are enough for the `host` section.

Reply:
No :). I largely used the pypi skeleton as a starting place, which included these dependencies. As I think about it, I think you are right. `interface_meta` need not ever be imported on the host, and so these deps are unnecessary. I'll patch it!